addons/crm/tests/test_crm_pls.py

```diff
@@ -32,6 +32,12 @@ def setUpClass(cls):
             'name': 'PLS Team',
         })
 
+        # Ensure independance on demo data
+        cls.env['crm.lead'].with_context({'active_test': False}).search([]).unlink()
+        cls.env['crm.lead.scoring.frequency'].search([]).unlink()
+        cls.env['crm.lead'].flush()
+        cls.env['crm.lead.scoring.frequency'].flush()
```

Comment:
can't you do ``cls.cr.flush()`` to flush everything at once ?

Reply:
It is less common,  but yes it makes sense here :) done